OneCommunityGlobal/HighestGoodNetworkApp
# Shereen_Bulk Reusables Update component

Added Bulk Reusable Update component.
Added reducer function.
Added action fyunction.
Added URL

# Description
Update Reusable - Bulk
Implements WBS # 10.2.5

## Related PRS (if any):
This frontend PR is related to the https://github.com/OneCommunityGlobal/HGNRest/pull/1005 backend PR.


…

## How to test:
1. check into current branch
3. do npm install and npm run start:local to run this PR locally
4. Clear site data/cache
5. Login as any user
6. Go to /bmdashboard (http://localhost:3000/bmdashboard) and log in again.
7. Navigate to [/bmdashboard/reusables/update] (http://localhost:3000/bmdashboard/reusables/update)
8.  Increment values in Used and Wasted columns and click Submit button. Check if the success message is coming after the submit button is clicked and New available column is updated accordingly.


## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/7041610/d6e4ebd6-6d5f-472f-8e1c-3c3d55a000df
